### PR TITLE
Fix: Moved JS logic into template jsx (fixes #14)

### LINKED ISSUE #14: Move title display / mobile hide logic into template
### Subject of the issue
The logic for displaying which title and for showing / hiding the title in mobile view could probably be moved into the template rather than being done in the JS.

### Your environment
* Master FW & AT

### PR DESCRIPTION
**Pull request by guywillis**
Fixes: #14 

### Fix
* Moved title display and show / hide mobile logic into template JSX
* Typo fix in contentObject schema

### CODE REVIEW

File: templates/navigationTitle.jsx [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
   } = props;
 
-  return (
+  const hideForMobile = ( _hideForMobile && !device.isScreenSizeMin('medium') )
+    ? false
+    : true;
+
+  const title = ( _useCourseTitle )
+    ? Adapt.course.get('title')
+    : Adapt.course.get('_navigationTitle').title;
```
Comment (swashbuck) [inline templates/navigationTitle.jsx]: I think this logic should probably go in the view/model rather than in the template?

Comment (oliverfoster) [reply templates/navigationTitle.jsx:17]: You can put it in here if you want?

https://github.com/cgkineo/adapt-navigationTitle/blob/17f41f7cc0296278225641a7b3dce93e02444d23/js/adapt-navigationTitle.js#L12-L14
```js
 static get courseConfig() { 
   const config = Adapt.course.get('_navigationTitle'); 
   const hideForMobile = ( _hideForMobile && !device.isScreenSizeMin('medium') )
    ? false
    : true;
   const title = ( _useCourseTitle )
    ? Adapt.course.get('title')
    : Adapt.course.get('_navigationTitle').title;
   return {
     ...config,
     hideForMobile,
     title
   };
 } 
```

Comment (guywillis) [reply templates/navigationTitle.jsx]: Moved title logic to JS

**Review by oliverfoster — APPROVED** [review]

File: templates/navigationTitle.jsx (hunk 1 above)
Comment (guywillis) [reply templates/navigationTitle.jsx]: @swashbuck I haven't moved the `hideForMobile` / `isVisible` logic to the view / model as I'm mirroring the approach in core:

https://github.com/adaptlearning/adapt-contrib-core/blob/master/templates/header.jsx#L38

**Review by swashbuck — APPROVED** [review]
👍

**Pull request merged**